testdriver/testdriver.py

```diff
@@ -18,11 +18,12 @@
 
 from testplan import TestPlan
 
+logger = logging.Logger("TEST DRIVER LOGGER")
+logger.setLevel(logging.WARNING)
```

Comment:
Unless there's a good reason, we should not manually (hardcode) the logger level in code. Instead, we should allow the logger level to be set by our existing config file. So here & other places in the PR, where you see `logger.setLevel(...)`, we should remove those statements.